src/metabase/db/setup.clj

```diff
@@ -74,7 +74,10 @@
     (log/info (trs "Setting up Liquibase..."))
     (liquibase/with-liquibase [liquibase conn]
       (try
-       (liquibase/consolidate-liquibase-changesets! conn)
+       (when (= :force direction)
```

Comment:
this is not needed here, since `force-migrate-up-if-needed!` will do the lock release.